specification/authorization/resource-manager/Microsoft.Authorization/preview/2018-05-01-preview/authorization-AccessReviewCalls.json

```diff
@@ -1679,6 +2147,165 @@
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The resource type."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RoleAssignmentApprovalProperties",
+          "description": "Role Assignment Approval properties."
+        }
+      },
+      "description": "Role Assignment Approval."
+    },
+    "RoleAssignmentApprovalProperties": {
+      "properties": {
+        "steps": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RoleAssignmentApprovalStep"
+          },
+          "description": "This is the collection of steps returned when one does an expand on it."
+        }
+      },
+      "description": "Role Assignment Approval properties."
+    },
+    "RoleAssignmentApprovalListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RoleAssignmentApproval"
+          },
+          "description": "Role Assignment Approval list."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to use for getting the next set of results."
+        }
+      },
+      "description": "List of role assignment approvals."
+    },
+    "RoleAssignmentApprovalStep": {
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The role assignment approval step id."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The role assignment approval step name."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The resource type."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RoleAssignmentApprovalStepProperties",
+          "description": "Role Assignment Approval Step properties."
+        }
+      },
+      "description": "Role assignment approval step properties."
+    },
+    "RoleAssignmentApprovalStepProperties": {
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The display name for the approval step."
+        },
+        "status": {
+          "type": "string",
+          "readOnly": true,
+          "description": "This read-only field specifies the status of an approval.",
+          "enum": [
+            "NotStarted",
+            "InProgress",
+            "Completed",
+            "Expired",
+            "Initializing",
+            "Escalating",
+            "Completing",
+            "Escalated"
+          ],
+          "x-ms-enum": {
+            "name": "RoleAssignmentApprovalStepStatus",
+            "modelAsString": true
+          }
+        },
+        "assignedToMe": {
```

Comment:
>assignedToMe [](start = 9, length = 12)

Is this not read only?

Reply:
Thank you for bringing this up. I've fixed this issue.